template.yaml

```diff
@@ -2,11 +2,25 @@ kind: Template
 apiVersion: v1
 template: spark
 metadata:
-  name: spark
+  name: apache-spark
+  creationTimestamp: null
+  annotations:
+    description: Apache Spark with 3 worker nodes.
+    iconClass: icon-nodejs
+    tags: quickstart,Apache,Spark,apachespark
 labels:
   app: sparkcluster
 objects:
-
+- apiVersion: v1
+  kind: PersistentVolumeClaim
```

Comment:
I was added to this thread by mistake. I believe you meant to add @tmckayus.

I might be way off base here, but instead of shared, persistent storage volumes, I believe most users of Apache Spark rely on HDFS.